Dockerfile

```diff
@@ -75,20 +56,14 @@ RUN mkdir -p /tmp/netlify && \
   && rm -rf /tmp/netlify /tmp/netlify.tar.gz \
   && netlify-deploy --help
 
-COPY --from=jx-release-version /usr/bin/jx-release-version /usr/bin/jx-release-version
-
 ## Repeating the ARGs from top level to allow them on this scope
```

Comment:
Not quite sure we have to keep this repetition now there isn't any COPY anymore.

Reply:
yup, you're right